wdl-lint/tests/lints/redundant-input-assignment-wdl-1.0/source.wdl

```diff
@@ -0,0 +1,19 @@
+#@ except: DescriptionMissing, MissingRequirements
+#@ except: RuntimeSectionKeys, MissingOutput, MissingMetas
+
+version 1.0
+
+workflow test {
+    input {
+        String arm
+        String cam
+        Int bam
+    }
+
+    # This should not flag any notes, since version is 1.0
+    call bar { input:
+        arm,
```

Comment:
@adthrasher do you know why this isn't causing an error? It shouldn't be allowed in 1.0 doc, or am I misunderstanding something in the spec?

Reply:
Shouldn't there be an error that `bar` is not defined?

Reply:
But yes, I would think that `arm` should trigger a parse error since it isn't allowed in 1.0.

Reply:
> Shouldn't there be an error that `bar` is not defined?

I think this is ok, but I need to code dive to verify. I believe that the undefined call check doesn't happen prior to our lint checks (only as part of `analysis` which these tests don't run).

> But yes, I would think that `arm` should trigger a parse error since it isn't allowed in 1.0.

This however is concerning to me. Going to do some investigating.

@michaelgattas you may have discovered a bug here. Not caused by your changes, but we should hold off merging till we know what's going on here.

Reply:
If this should have thrown an error, then when everything works as expected, this test file should be modified so that it doesn't indicate an error, right?

Reply:
We'll cross that bridge when we get to it. I actually want to leave the error in because I want to see it trip when that bug gets addressed.